src/main/kotlin/com/hedvig/underwriter/serviceIntegration/lookupService/LookupServiceImpl.kt

```diff
@@ -52,12 +57,14 @@ class LookupServiceImpl(
             logger.info("LivingArea($livingAreaMatches) or postalCode($postalCodeMatches) matches, so we're assuming this is the same object")
             return true
         } else {
+            val streetAddressMatches = streetAddressMatches(quote, response);
+
             // Do some logging only, so we can evaluate
-            if (streetAddressMatches(quote, response)) {
-                logger.info("(Test) Street and number matched, but not postalCode/livingArea. quoteId: ${quote.id}, dataCollectionId: ${quote.dataCollectionId}")
+            if (streetAddressMatches) {
+                logger.info("(Test) Street and number matched, but not postalCode/livingArea(probably missing data). quoteId: ${quote.id}, dataCollectionId: ${quote.dataCollectionId}. Will trust it: $useInsurelyStreetAddress")
```

Comment:
Shall we remove the test part? 
```suggestion
                logger.info("Street and number matched, but not postalCode/livingArea(probably missing data). quoteId: ${quote.id}, dataCollectionId: ${quote.dataCollectionId}. Will trust it: $useInsurelyStreetAddress")
```

Reply:
Good catch!